orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/StartScriptTask.groovy

```diff
@@ -52,6 +53,14 @@ class StartScriptTask implements Task {
     String cluster = stage.context.cluster
     String cmc = stage.context.cmc
 
+    if(stage.execution instanceof Pipeline && stage.execution.trigger.parameters?.strategy == true){
+      Map trigger = ((Pipeline) stage.execution).trigger
+      image = image ?: trigger.parameters.amiName ?: trigger.parameters.imageId ?: ''
+      cluster = cluster ?: trigger.parameters.cluster ?: ''
+      account = account ?: trigger.parameters.credentials ?: ''
+      region = region ?: trigger.parameters.region ?: trigger.parameters.zone ?: ''
```

Comment:
We really need to centralize the region vs. zone somewhere sometime. The task level is almost certainly too granular? (funny enough, @ttomsu and I were just talking about this).